github/ipfs.yml

```diff
@@ -5836,11 +5836,11 @@ repositories:
     teams:
       admin:
         - admin
-      pull:
-        - github-mgmt stewards
-      push:
         - ipdx
+        - kubo maintainers
         - w3dt-stewards
```

Comment:
LGTM, but FYI it looks like the kubo-maintainers group is a strict subset of the w3dt-stewards group. Did you mean to add both?

Side note: w3dt-stewards is probably too big (and of course is a highly dated name) but can tackle that in another PR.

Reply:
yeah, i blindly replicated what we already did in other places. 
cleaning up these teams will be something for a future PR.